packages/gatsby-theme-docs/src/components/video-player.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LoadingSpinner from '@commercetools-uikit/loading-spinner';
```

Comment:
Concerning the loading UX, we don't have to get it 100% perfect but the current preview has too much layout shift and general "weirdness".   Can you at least take a look at the quiz component, which is already doing something similar 
- reserve the space before the client component is loaded based on the assumption that it's a 16:10 video in full responsive width - maybe frame it with a grey hairline 
- center the loading animation. 
- consider using a 16:10 aspect ratio example video.  We will only very unlikely use a wide format like that example you used, rather more squarish than the other direction.

Reply:
@nkuehn I tried to do that but I'm not 100% sure to understand where I should place the container. I still see quite a lot of layout shift. Any ideas?

Reply:
I think I now got it right